data-types.md

```diff
@@ -5,11 +5,11 @@ This MDS data types page catalogs the objects (fields, types, requirements, desc
 ## Table of Contents
 
 - [Vehicles](#vehicles)
-  - [Vehicle Types](#vehicle-types)
```

Comment:
This link should remain as an anchor with the #vehicle-types link. It doesn't need to be a shortcut, as it's in a Table of Contents.